File: 240522/배열의 특정 위치 찾기/find-specific-location-fo-array.py
arr = list(map(int, input().split()))

sum_2 = 0
sum_3 = 0
cnt = 0

for i in range(1, len(arr), 2):
    sum_2 += arr[i]

for j in range(2, len(arr), 3):
    cnt += 1
    sum_3 += arr[j]

print(sum_2, sum_3/cnt)

File: 241223/배열의 특정 위치 찾기/find-specific-location-fo-array.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int arr[10], sum_even=0, sum3=0, cnt=0;
    for (int i=0; i<10; i++) {
        cin >> arr[i];
        if (i%2 == 1) {
            sum_even += arr[i];
        }
        if ((i+1)%3 == 0) {
            sum3 += arr[i];
            cnt++;
        }
    }
    cout << fixed;
    cout.precision(1);
    cout << sum_even << ' ' << (double)sum3/cnt;
    return 0;
}